### ISSUE TITLE
Reproduction on the AMASS Dataset

### ISSUE THREAD

**Issue by Deante-dx** [dulucas/siMLPe#10]
I have been attempting to replicate the results on AMASS reported in your paper. However, I have encountered some challenges and discrepancies while trying to reproduce the reported outcomes (10.8 19.6 34.3 40.5 50.5 57.3 62.4 65.7). I have (10.9, 19.8, 34.9, 41.1, 51.2, 58.1, 63.1, 66.8)
I would greatly appreciate your guidance on this matter. Could you kindly provide any insights or suggestions as to why I might be experiencing these discrepancies?

**Comment by Logan-007L** [comment]
Hello, could you tell me which amass data sets I need? All of them? Or just pick some of them such as ACCAD?

**Comment by dulucas** [comment]
> I have been attempting to replicate the results on AMASS reported in your paper. However, I have encountered some challenges and discrepancies while trying to reproduce the reported outcomes (10.8 19.6 34.3 40.5 50.5 57.3 62.4 65.7). I have (10.9, 19.8, 34.9, 41.1, 51.2, 58.1, 63.1, 66.8) I would greatly appreciate your guidance on this matter. Could you kindly provide any insights or suggestions as to why I might be experiencing these discrepancies?

Sorry I can't recall the details as it has been a while. It may due to the randomness of the exp(the hardware/environment/seed etc). I did recall that sometimes reload the model from 80000 or 100000 iters w/o optimizer and then do the finetuning with lower lr could help.

**Comment by dulucas** [comment]
> Hello, could you tell me which amass data sets I need? All of them? Or just pick some of them such as ACCAD?

We use a split of AMASS dataset following the setting of previous pubs. [Here](https://github.com/dulucas/siMLPe/tree/main/data) you can find the train/test split for AMASS.
